contact.php

```diff
@@ -0,0 +1,61 @@
+<?php include'head.php';
+require_once'bd.php';
+
+$reponse=$bdd->query("SELECT * FROM contact ORDER BY id DESC");
+
+if(isset($_POST['submit'])){
+    $name1=$_POST['fname'];
+    $name2=$_POST['lname'];

```

Comment:
I'd name the variables `first_name` and `last_name` otherwise you might get confused with the numbers.